contrib/completion/bash/docker

```diff
@@ -874,6 +875,7 @@ __docker_complete_capabilities_droppable() {
 		MKNOD
 		NET_BIND_SERVICE
 		NET_RAW
+		RESET
 		SETFCAP
```

Comment:
I noticed that our completion scripts use the capabilities without the `CAP_` prefix; I'm on the fence what we should use as "normalised" format: with, or without the `CAP_`. From a UX perspective, possibly _without_ is more friendly (less to type, and less characters to type to use auto-completion)

@tiborvass @silvin-lubecki WDYT?

Reply:
Perhaps it doesn't make a big difference (as the values entered by the user will be normalised anyway, so the only difference will be in what's shown in `docker service inspect`. Looks like that's currently consistent with (e.g.) what's documented for `docker plugin` capabilities.

Reply:
I'm totally fine with values without the `CAP_` 👍

Reply:
When completing signals, we offer both SIGXXX and XXX.
So for consistency, we could switch to supporting both variants here as well.
I'll create a PR for that.

Reply:
Created #2706